CHANGELOG.md

```diff
@@ -55,8 +57,7 @@ _No breaking changes!_
 - Updated `Dry::Core` gem version to 1.0.0 in benchmarks [[#297]](https://github.com/panorama-ed/memo_wise/pull/297)
```

Comment:
Feel free to say no, but while I have you here I just noticed I'd messed up some of the changelog link stylings in the v1.9.0 and v1.8.0 sections (the second `]` should come after the `)`). Would you mind fixing those?